core/src/main/resources/hudson/AboutJenkins/index.jelly

```diff
@@ -25,44 +25,47 @@ THE SOFTWARE.
 <!-- 3rd party license acknowledgements and  -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:s="/lib/form">
-  <l:layout title="${%about(app.VERSION)}" norefresh="true">
-    <l:main-panel>
-      <h1 version="${app.VERSION}">${%about(app.VERSION)}</h1>
-      <p>
-        ${%blurb}
-      </p>
 
-      <p>
-        ${%dependencies}
-      </p>
-      <j:set var="uri" value="${it.licensesURL}"/>
-      <j:choose>
-          <j:when test="${uri != null}">
+  <l:layout norefresh="true" type="one-column" title="${%about(app.VERSION)}">
+
+    <l:main-panel>
```

Comment:
I tried to do that but when you add DOM modifications is easy to reach out a lot of indentation changes, etc... The approach is the same than https://github.com/jenkinsci/jenkins/pull/2029